<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>693</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>621</width>
      <height>661</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>250Hz</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>601</width>
        <height>621</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="1" column="0">
        <widget class="QPushButton" name="read">
         <property name="text">
          <string>读取</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="Slider" name="horizontalSlider">
         <property name="tracking">
          <bool>false</bool>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QScrollArea" name="scrollAreaWidgetContents">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_7">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>597</width>
            <height>506</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_14">
           <item>
            <widget class="QCustomPlot" name="customplot_preview" native="true"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_29">
           <item>
            <widget class="QLabel" name="label_1">
             <property name="text">
              <string>读取磁盘：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>数据帧数目：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>数据头字节数：</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_30">
           <item>
            <widget class="QLineEdit" name="address">
             <property name="text">
              <string>F</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="times">
             <property name="text">
              <string>50</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="headnum">
             <property name="text">
              <string>6</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_31">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>偏移地址：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>数据帧字节数：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>有效数据数：</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_32">
           <item>
            <widget class="QLineEdit" name="offset">
             <property name="text">
              <string>14680064</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="datanum">
             <property name="text">
              <string>4096</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="effnum">
             <property name="text">
              <string>4080</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>8000Hz</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>601</width>
        <height>621</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="1" column="0">
        <widget class="QPushButton" name="read_2">
         <property name="text">
          <string>读取</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="Slider" name="horizontalSlider_2">
         <property name="tracking">
          <bool>false</bool>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QScrollArea" name="scrollAreaWidgetContents_2">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_8">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>597</width>
            <height>506</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_15">
           <item>
            <widget class="QCustomPlot" name="customplot_preview_2" native="true"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_16">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_33">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>读取磁盘：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>数据帧数目：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>数据头字节数：</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_34">
           <item>
            <widget class="QLineEdit" name="address_2">
             <property name="text">
              <string>F</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="times_2">
             <property name="text">
              <string>50</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="headnum_2">
             <property name="text">
              <string>6</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_35">
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>偏移地址：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>数据帧字节数：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>有效数据数：</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_36">
           <item>
            <widget class="QLineEdit" name="offset_2">
             <property name="text">
              <string>16101933056</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="datanum_2">
             <property name="text">
              <string>4096</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="effnum_2">
             <property name="text">
              <string>4086</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>635</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Slider</class>
   <extends>QSlider</extends>
   <header>slider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
